### PR TITLE
XW-3821 | Add new URLs to app buttons in global footer + update translations

### PR DESCRIPTION
**Pull request by xkxd**
https://wikia-inc.atlassian.net/browse/XW-3821

@Wikia/x-wing 

## NOTE!

Regarding multiple changes in i18n jsons - it happened because of running `npm run setup` in DesignSystem's directory and it related to changing the order of keys to mirror the one from design-system-i18n repo - I can revert that if it's necessary.

### CODE REVIEW

**Review by rybmat — APPROVED** [review]

File: includes/wikia/models/DesignSystemGlobalFooterModel.class.php [hunk 1]
```diff
@@ -159,12 +159,12 @@ public function getData() {
 					'type' => 'line-text',
 					'title' => [
 						'type' => 'translatable-text',
-						'key' => 'global-footer-community-apps-header'
+						'key' => 'global-footer-fandom-app-header'
```
Comment (latata) [inline includes/wikia/models/DesignSystemGlobalFooterModel.class.php]: What about the case that `The FANDOM App` string is only for english?

Comment (xkxd) [reply includes/wikia/models/DesignSystemGlobalFooterModel.class.php]: I'm aware of that, this was submitted before we agreed to change the ACs

File: extensions/wikia/DesignSystem/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "setup-template": "npm install && grunt"
   },
   "dependencies": {
-    "design-system-i18n": "Wikia/design-system-i18n#1.0.0",
+    "design-system-i18n": "Wikia/design-system-i18n#XW-4018",
```
Comment (latata) [inline extensions/wikia/DesignSystem/package.json]: remember to replace that after merging translations

**Review by latata — APPROVED** [review]

**Review by latata — CHANGES_REQUESTED** [review]
Update ds-i18n version

**Review by latata — APPROVED** [review]

**Pull request merged**